types.d.ts

```diff
@@ -56,6 +56,8 @@ interface Window {
         }
     }
 
+    providerStatusUpdated: (data: import('./src/deviceApiCalls/__generated__/validators-ts').ProviderStatusUpdated) => void;
```

Comment:
I've learned the hard way recently that any `import` in a `.d.ts` file does not give the safety you might assume. For example if you make a typo I think this would default to Any.

I wonder if `ProviderStatusUpdated` would work on it's own? if not, don't consider this comment a blocker at all - I'm just curious about improving this 🦶🔫

Reply:
It does work, but it doesn't change that if there's a typo it falls back to `any`.